projects/valtimo/config/assets/core/en.json

```diff
@@ -363,7 +363,9 @@
     "of": "of",
     "visible": "visible",
     "next": "Next >",
-    "previous": "< Previous"
+    "previous": "< Previous",
+    "addRow": "Add row",
```

Comment:
For some reason the FE still shows a button with: 'interface.addRow'

Reply:
Probably some caching / build issue